hotel-management/src/components/Toast/Toast.tsx

```diff
@@ -0,0 +1,32 @@
+import { useEffect } from 'react';
+
+import { ToastItem } from './type';
+import './index.css';
+
+type ToastProps = ToastItem & {
```

Comment:
This extend is okay but what if ToastItem has some props that redundant in this particular component